src/Enum.php

```diff
@@ -215,6 +225,7 @@ public static function isValid($value)
      *
      * @psalm-pure
      * @psalm-assert T $value
+     * @param mixed $value
```

Comment:
Why add this parameter docblock?

Reply:
There is a `MissingParamType` psalm error.
https://psalm.dev/docs/running_psalm/issues/MissingParamType/